### PR TITLE
feat: add "Sync Now" to Plaid Settings

### PR DESCRIPTION
**Pull request by Alchez**
Currently, Plaid transactions are synced up hourly, but there's no way to trigger a sync in realtime.

### CODE REVIEW

File: erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.js [hunk 1]
```diff
@@ -15,6 +15,21 @@ frappe.ui.form.on('Plaid Settings', {
...
 			});
+
+			frm.add_custom_button(__("Sync Now"), () => {
+				frappe.call({
+					method: "erpnext.erpnext_integrations.doctype.plaid_settings.plaid_settings.enqueue_synchronization",
+					freeze: true,
+					callback: () => {
+						frappe.msgprint({
+							title: __("Sync Started"),
+							message: __(`The sync has started in the background, please check the
```
Comment (rmehta) [inline erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.js]: translatable strings can't be multi-line

**Review by rmehta — CHANGES_REQUESTED** [review]

File: erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.js (hunk 1 above)
Comment (Alchez) [reply erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.js]: Changed the string format

**Review by nextchamp-saqib — DISMISSED** [review]

File: erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.js [hunk 2]
```diff
@@ -15,6 +15,22 @@ frappe.ui.form.on('Plaid Settings', {
...
+					method: "erpnext.erpnext_integrations.doctype.plaid_settings.plaid_settings.enqueue_synchronization",
+					freeze: true,
+					callback: () => {
+						let bank_transaction_link = '<a href="#List/Bank Transaction">Bank Transaction</a>';
+
+						frappe.msgprint({
+							title: __("Sync Started"),
+							message: __("The sync has started in the background, please check the {0} list for new records.", [bank_transaction_link]),
+							alert: 1
+						})
```
Comment (nextchamp-saqib) [inline erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.js]:
```suggestion
						});
```

File: erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.js [hunk 3]
```diff
@@ -15,6 +15,22 @@ frappe.ui.form.on('Plaid Settings', {
...
+					}
+				})
```
Comment (nextchamp-saqib) [inline erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.js]:
```suggestion
				});
```

**Pull request merged**